<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MSI" failures="1" tests="1" name="Default test" time="1.812" errors="0" timestamp="2024-01-11T09:55:54 IST">
  <testcase classname="TestBase.BaseClass" name="@BeforeClass setup" time="-1.704947152782E9">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;os&amp;apos; is required by BeforeClass on method setup but has not been marked @Optional or defined
in C:\Users\91814\AppData\Local\Temp\testng-eclipse--877558403\testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'os' is required by BeforeClass on method setup but has not been marked @Optional or defined
in C:\Users\91814\AppData\Local\Temp\testng-eclipse--877558403\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:267)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:355)
at org.testng.internal.Parameters.createParameters(Parameters.java:684)
at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:166)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:306)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:180)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setup -->
  <testcase classname="TestBase.BaseClass" name="@AfterClass teardown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass teardown -->
  <testcase classname="TestCases.TC_002_logginDDT" name="test_logDDT" time="1.762">
    <failure type="org.testng.internal.reflect.MethodMatcherException" message="
[public void TestCases.TC_002_logginDDT.test_logDDT(java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: test_logDDT([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2])
Arguments: [(java.lang.String) jithendranath@gmail.com,(java.lang.String) jith.417,(java.lang.String) valid,(java.lang.String) ,(java.lang.String) ,(java.lang.String) ]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void TestCases.TC_002_logginDDT.test_logDDT(java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: test_logDDT([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2])
Arguments: [(java.lang.String) jithendranath@gmail.com,(java.lang.String) jith.417,(java.lang.String) valid,(java.lang.String) ,(java.lang.String) ,(java.lang.String) ]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:909)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:37)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test_logDDT -->
</testsuite> <!-- Default test -->
